build/ci/cloudbuild.push.yaml

```diff
@@ -73,6 +73,28 @@ steps:
       - |
         ./script/update_staging_version.sh $SHORT_SHA
 
+  # This is a trigger to push website at website/mixer code HEAD and with the
+  # latest base cache.
+  - name: 'gcr.io/datcom-ci/deploy-tool'
+    entrypoint: 'bash'
+    args:
+    - '-c'
+    - |
+      set -e
+      mkdir src
+      git clone https://github.com/datacommonsorg/website.git src
+      cd src
+      git clone https://github.com/datacommonsorg/mixer.git mixer
+      git submodule update --init --recursive
+      cd mixer
+      git remote set-url origin https://github.com/datacommonsorg/mixer.git
+      git pull origin master
+      git checkout master
+      gsutil cp gs://automation_control/latest_base_cache_version.txt deploy/storage/bigtable.version
```

Comment:
wondering about bigquery version... website doesn't use it so this is ok.

but is there is a separate mixer version of this? that might need bigquery

Reply:
I think in general would be helpful to have a bigquery version file in gcs as well. Can add it here.

Mixer does not have autopush yet

Reply:
Ok, I'll make the change to write bigquery file.